prow/plugins.yaml

```diff
@@ -356,6 +356,11 @@ plugins:
   - trigger
 
   kubernetes/cloud-provider-azure:
+  - milestone
+  - milestonestatus
```

Comment:
I won't hold for this, but this should go away.  Opened https://github.com/kubernetes/test-infra/issues/11037

Reply:
we can remove it when we clean up #11037 😅